src/js/main.js

```diff
@@ -600,6 +600,18 @@ Site.Map = {
           }
         });
 
+        if(WP.wp_debug == true && WP.isAdmin == true) {
```

Comment:
should this condition just be an or? so they can see on the live site if there are duplicates?